flags.go

```diff
@@ -336,15 +336,6 @@ func newApp() (app *cli.App) {
 				Name:  "debug_mutex",
 				Usage: "Print debug messages when a mutex is held too long.",
```

Comment:
we should also update doc of debug_http flag to say that "The flag is currently used" . This is because with this deprecation, debug_http will not be functional anymore.